EIPS/eip-4626.md

```diff
@@ -470,6 +467,8 @@ Those methods should be performed separately.
 
 `sender` has exchanged `assets` for `shares`, and transferred those `shares` to `receiver`.
 
+`assets` MUST match the increase of `totalAssets`, and `shares` MUST match the increase of `totalSupply`.
```

Comment:
Not sure I'm onboard with this, because then it doesn't account for deposit/withdraw fees properly for comparison's sake. A portion of `assets` is transferred to another address to represent those fees, which is reducing the overall rate of return you should expect to receive.

Reply:
Same, we should remove both of these clarifications on the events